manim/mobject/svg/brace.py

```diff
@@ -140,16 +140,18 @@ def __init__(
         brace_direction=DOWN,
         label_constructor=MathTex,
         label_scale=1,
+        buff=0.2,
         **kwargs
     ):
         self.label_constructor = label_constructor
         self.label_scale = label_scale
         super().__init__(**kwargs)
 
         self.brace_direction = brace_direction
+        self.buff = buff
```

Comment:
Do we need to store it here? Seems like it is just passed straight through to Brace

Reply:
ye not needed but added that just in case for future, idk